.circleci/config.yml

```diff
@@ -89,7 +89,7 @@ commands:
         default: ''
       cargo_behavior:
         type: string
-        default: --locked --offline --verbose
```

Comment:
Drop this for now, we should fix the CI separately and @vincenthz said he has something for this.